### DISCUSSION TITLE
Release date for 1.3.42

### DISCUSSION THREAD

**Discussion by RadekWikturna** [LibrePDF/OpenPDF#1083] [General]
Is there any plan to release 1.3.42 soon? When? I'm asking because I've contributed some fixes so that we could start using OpenPdf in our project instead of iText.

**Comment by asturio** [comment]
Just working in the latest PR, so Java 8 compatibility is preserved. Won't take long.

**Comment by RadekWikturna** [comment]
Version 1.3.42 has just appeared on https://mvnrepository.com/artifact/com.github.librepdf/openpdf/1.3.42
